README.md

```diff
@@ -196,7 +196,7 @@ The `dfe:analytics:install` generator will also initialize some empty config fil
 | `config/analytics_blocklist.yml`      | Autogenerated file to list all fields we will NOT send to BigQuery, to support the `analytics:check` task                                                                                              |
 | `config/analytics_custom_events.yml`  | Optional file including list of all custom event names                                                             |
 
-**It is imperative that you perform a full check of the fields that are being sent, and exclude those containing personally-identifiable information (PII) in `config/analytics_hidden_pii.yml`, in order to comply with the requirements of the [Data Protection Act 2018](https://www.gov.uk/data-protection), unless an exemption has been obtained.**
+**It is imperative that you perform a full check of the fields that are being sent, exclude those for which you have no legal basis for processing under GDPR in `config/analytics_blocklist.yml`, and exclude those containing personally-identifiable information (PII) in `config/analytics_hidden_pii.yml`, in order to comply with the requirements of the [Data Protection Act 2018](https://www.gov.uk/data-protection), unless an exemption has been obtained. Further guidance on this is available [here](./docs/deciding_what_data_to_stream_with_dfe_analytics.md).**
```

Comment:
If there are PII fields in the database, then the advise here is to add these to `config/analytics_hidden_pii.yml`. If we are not interested in these fields at all then should we advise to add them to `config/analytics_blocklist.yml` ? I expect we can't just ignore them altogether as analytics will throw an error ?

Reply:
I've clarified this. I think the bigger distinction users need to draw is between PII and human readable PII; all data they're streaming should be treated as PII.